djangocms_navigation/admin.py

```diff
@@ -24,6 +20,15 @@
 from .views import ContentObjectSelect2View, MenuContentPreviewView
 
 
+# TODO: Tests to be added (FIL-908).
```

Comment:
We can't really include FIL numbers when submitting to Divio repo

Reply:
FIL number is supposed to be in the FIL pull request (but not the divio pull request)... makes it hard for me to know which story this PR relates to.

Reply:
If you double check, this branch has FIL number of the current story number. I've added FIL number on TODO task as work will be done as part of that story number. Navigation is Fidelity specific repo and not the Divio Repo. However, you still want me to remove it please let me know.